test/job.js

```diff
@@ -672,7 +672,7 @@ describe('Job', () => {
 
       jobs.define('lock job', {
         lockLifetime: 50
-      }, () => {
+      }, (job, cb) => { // eslint-disable-line no-unused-vars
```

Comment:
@simison I think I found the reason this is failing. Even if two params aren't used they need to be there for this test to pass otherwise it only gets called once.